### ISSUE TITLE
testing fails because of absent include

### ISSUE THREAD

**Issue by cookie33** [EUDAT-B2SAFE/B2HANDLE#38]
Hi,

The tests fail:

```
$ python main_test_script.py
Traceback (most recent call last):
  File "main_test_script.py", line 15, in <module>
    from handleconnector_writeaccess_patched_test import EUDATHandleConnectorWriteaccessPatchedTestCase
ImportError: No module named handleconnector_writeaccess_patched_test

$ grep -i handleconnector_writeaccess_patched_test *
main_test_script.py:from handleconnector_writeaccess_patched_test import EUDATHandleConnectorWriteaccessPatchedTestCase
grep: resources: Is a directory
grep: testcredentials: Is a directory

$ git branch -va
* devel                     a18edd9 Merge pull request #35 from merretbuurman/searchout
  master                    17aacb2 Merge pull request #26 from EUDAT-B2SAFE/devel
  remotes/origin/HEAD       -> origin/master
  remotes/origin/devel      a18edd9 Merge pull request #35 from merretbuurman/searchout
  remotes/origin/gh-pages   ef64412 Add first version of Sphinx docs
  remotes/origin/master     17aacb2 Merge pull request #26 from EUDAT-B2SAFE/devel
  remotes/upstream/devel    a18edd9 Merge pull request #35 from merretbuurman/searchout
  remotes/upstream/gh-pages 6955841 Update docs
  remotes/upstream/master   17aacb2 Merge pull request #26 from EUDAT-B2SAFE/devel
```

Greetings,

Robert

**Comment by NicolasLiampotis** [comment]
Fails for me too. The build server does not complain because `handleconnector_writeaccess_patched_test.pyc` is still available from previous build.

**Comment by NicolasLiampotis** [comment]
Fixed via PR #39

**Comment by cookie33** [comment]
Hi,

It fails differently now on python 2.6.

```
$ python main_test_script.py
Specified test types: ['unit']

Collecting tests:
Number of tests for client (no access required):                                24
Number of tests for handle system connector (no access required):                               11
Number of tests for PIDClientCredentials:                                       16
Number of tests for searching (without server access):                          9
Number of tests for client (faked read access):                                 15
Number of tests for client's 10320/LOC (faked read access):                     7
Number of tests for patched read access:                                        17
Number of tests for patched write access:                                       42
Traceback (most recent call last):
  File "main_test_script.py", line 137, in <module>
    patched_write_conn = unittest.TestLoader().loadTestsFromTestCase(EUDATHandleConnectorWriteaccessPatchedTestCase)
NameError: name 'EUDATHandleConnectorWriteaccessPatchedTestCase' is not defined
```

Robert

**Comment by NicolasLiampotis** [comment]
Really fixed via PR #40
